src/ingestors/rarible/offchain-metadata.ts

```diff
@@ -0,0 +1,45 @@
+export const urlForValidRaribleCollection = async (
+    chainId: number,
+    contractAddress: string,
+): Promise<string | undefined> => {
+    const apiKey = process.env.RARIBLE_API_KEY;
+    if (!apiKey) {
+        throw new Error('RARIBLE_API_KEY is not defined in the environment variables');
```

Comment:
Deployment note -- requires `RARIBLE_API_KEY` to be set.

Confirming availability of this key.